test/src/e2e.dynval/dv.n'.test.ts

```diff
@@ -0,0 +1,375 @@
+            10, // minDecomposeAssetCost
+            10, // minUnwrapCccCost
+            4194304, // maxBodySize
+            16384, // snapshotPeriod
+            TERM_SECONDS, // termSeconds
+            10, // nominationExpiration
+            10, // custodyPeriod
+            30, // releasePeriod
+            8, // maxNumOfValidators
+            4, // minNumOfValidators
+            1000, // delegationThreshold
+            10000, // minDeposit
+            128 // maxCandidateMetadataSize
+        ];
+        const changeParams: [
+            number,
+            number,
+            (number | string)[],
+            ...string[]
+        ] = [0xff, 0, newParams];
+        const message = blake256(RLP.encode(changeParams).toString("hex"));
+        changeParams.push(`0x${SDK.util.signEcdsa(message, faucetSecret)}`);
+        changeParams.push(`0x${SDK.util.signEcdsa(message, aliceSecret)}`);
+        changeParams.push(`0x${SDK.util.signEcdsa(message, bobSecret)}`);
+
+        return await node.sdk.rpc.chain.sendSignedTransaction(
+            node.sdk.core
+                .createCustomTransaction({
+                    handlerId: stakeActionHandlerId,
+                    bytes: RLP.encode(changeParams)
+                })
+                .sign({
+                    secret: faucetSecret,
+                    seq: await node.sdk.rpc.chain.getSeq(faucetAddress),
+                    fee: 10
+                })
+        );
+    }
+
+    async function waitForTermChange(target: number, timeout?: number) {
+        const start = Date.now();
+        while (true) {
+            const termMetadata = await stake.getTermMetadata(
+                otherDynNodes[0].sdk
+            );
+            if (termMetadata && termMetadata.currentTermId >= target) {
+                break;
+            }
+            await wait(1000);
+            if (timeout) {
+                if (Date.now() - start > timeout * 1000) {
+                    throw new Error(`Term didn't changed in ${timeout} s`);
+                }
+            }
+        }
+    }
+
+    beforeEach(async function() {
+        await bootstrap();
+    });
+
+    describe("1. Jail one of the validator + increase the delegation of a candidate who doesn’t have enough delegation", async function() {
+        it("Alice should get out of the committee and Betty should be included in the committee", async function() {
+            // Kill the alice node first to make alice not to participate in the term 1.
+            await aliceNode.clean();
+
+            // enable!
+            const changeTx = await changeTermSeconds(initialNodes[0]);
+            for (let i = 0; i < otherDynNodes.length; i++) {
+                await otherDynNodes[i].waitForTx(changeTx);
+            }
+
+            const margin = 1.1;
+
+            await waitForTermChange(1, TERM_SECONDS * margin);
+
+            {
+                const authors = await otherDynNodes[0].sdk.rpc.sendRpcRequest(
```

Comment:
`const authors = await otherDynNodes[0].sdk.rpc.chain.getPossibleAuthors();`

Reply:
`getPossibleAuthors` not in the current SDK version.

Reply:
How about using `stake.getPossibleAuthors`?

Reply:
I updated!.